.github/workflows/deny.yml

```
@@ -13,4 +13,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
-    - uses: EmbarkStudios/cargo-deny-action@v2
+    - uses: EmbarkStudios/cargo-deny-action@v2.0.5
```

```suggestion
    - uses: EmbarkStudios/cargo-deny-action@v2
```
I pinned this to check if there was no regression in the action itself, I guess we should be good with minor updates.